### DISCUSSION TITLE
Implementation of Win10X Taskbar

### DISCUSSION THREAD

**Discussion by BillionXP** [valinet/ExplorerPatcher#2969] [Ideas]
A good feature in the future would be to add the windows 10x looking taskbar in the taskbar choice menu. Having windows 10 functionality

**Comment by pyrates999** [comment]
Try this: https://www.windowslatest.com/2020/02/23/get-windows-10xs-centered-taskbar/

**Comment by haydengriz** [comment]
Would be neat to have the windows 10x button
